### ISSUE TITLE
Feature Proposal: Form content filters

### ISSUE THREAD

**Issue by AtjonTV** [getgrav/grav-plugin-form#273]
## The Issue:
In the past view days I got a **lot** of spam through a contact form on my website. I get about three spam mails a day.

## How to fix?
It would be great if you could manage filters in the Plugin settings for Emails and form content. If I had this filtering available I could enter the key words that are in all of those spam mails to prevent future messages. All of those spam mails contain the keywords `cashnow.cars`, `Loan` and `Payday`.

## How to implement?
It would be great if you can configure a set of filters in the plugin settings page and load them into each form separately. Like:

```yaml
form:
 filters: filter1, filter2
 name: contact-form
...
```

## Is this even possible?
Yes it is. But its just a matter of time to get it done as it requires to create a new settings page to manage the filters, adding the parsing of the `filters` tag within the forms as well as making the filtering system itself.

----
In the case you have more questions about what I mean or how I would like it to be and I don't respond to your massage on Github, contact me at admin.atjontv@atvg-studios.at and inform me about your message.

PS: Thanks to every dev at Grav for reading my proposal!

**Comment by AtjonTV** [comment]
Yea .. I am using Captcha but it doesn't help at all.

I didn't knew there was something like a Honeypot field .. but I added one now. I don't think that the Honeypot would help as I already got Captcha, and spam is still comming.

**Comment by rhukster** [comment]
are you 'sure' that you are getting spam through your captcha protected form and not somewhere else?  Google captcha is very reliable, is it setup and working correctly?

**Comment by AtjonTV** [comment]
Yes. If I enter data but don't fill the captcha, I get the alert to fill in the Captcha.

The form is called "contact-form" and I can see it in the data-manager. And only it. There is no other form anywhere on the site.
